src/libs/PolicyUtils.js

```diff
@@ -133,6 +133,23 @@ function getClientPolicyMemberEmailsToAccountIDs(policyMembers, personalDetails)
     return memberEmailsToAccountIDs;
 }
 
+function getExcludedUsers(policyMembers, personalDetails) {
+    const memberEmailsToExclude = [...CONST.EXPENSIFY_EMAILS];
+    _.each(policyMembers, (policyMember, accountID) => {
+        // Policy members that are pending delete or have errors are not valid and we should show them in the invite options (don't exclude them).
+        if (policyMember.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE || !_.isEmpty(policyMember.errors)) {
+            return;
```

Comment:
JS Doc is missing here

Reply:
@s-alves10 Can you fix this alone, please?

Reply:
done